cfgov/v1/models/base.py

```diff
@@ -549,19 +569,24 @@ def too_many_attempts(self, value, timestamp):
         attempts = self.failed_attempts.split(',')
         return len(attempts) > value
 
+
 class TemporaryLockout(models.Model):
     user = models.ForeignKey(User)
     created = models.DateTimeField(auto_now_add=True)
     expires_at = models.DateTimeField()
 
 
 class Feedback(models.Model):
-    is_helpful = models.BooleanField()
-    comment = models.TextField(blank=True, null=True)
+    submitted_on = models.DateTimeField(auto_now_add=True)
     page = models.ForeignKey(
         Page,
         related_name='feedback',
         null=True,
         on_delete=models.SET_NULL,
     )
-    submitted_on = models.DateTimeField(auto_now_add=True)
+    comment = models.TextField(blank=True, null=True)
+    referrer = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
What purpose does the `referrer` field serve that the `page` field doesn't? Is it for pages that aren't in Wagtail?

FWIW when this was first being implemented [I thought](https://github.com/cfpb/cfgov-refresh/pull/2418#discussion_r80705807) it might be better to just use the page URL or slug for `page` instead of the foreign key to a Wagtail `Page` object. I would be okay with removing that `page` field altogether and always just storing the referrer. Do we know anything about what the requirements are for storing where feedback comes from on those legacy pages?